### PR TITLE
bluestore/BlueFS: pass string as const ref

### PR DESCRIPTION
**Pull request by dingdangzhang**
Signed-off-by: dingdangzhang <boqian.zy@alibaba-inc.com>
ps:i'm so sorry.the previous commit was lost due to fault operation.so commit again

### CODE REVIEW

File: src/os/bluestore/BlueFS.h [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+src/os/bluestore/BlueFS.cc// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```
Comment (tchaikov) [inline src/os/bluestore/BlueFS.h]: this does not compile.

Comment (dingdangzhang) [inline src/os/bluestore/BlueFS.h]: Has been modified

File: src/msg/async/Stack.cc [hunk 2]
```diff
@@ -115,8 +115,10 @@ NetworkStack::NetworkStack(CephContext *c, const string &t): type(t), started(fa
 
   for (unsigned i = 0; i < num_workers; ++i) {
     Worker *w = create_worker(cct, type, i);
-    w->center.init(InitEventNumber, i, type);
-    workers.push_back(w);
+    if (nullptr != w) {
```
Comment (tchaikov) [inline src/msg/async/Stack.cc]: `abort()` is called if `create_worker()` fails.

Comment (dingdangzhang) [inline src/msg/async/Stack.cc]: yes, close this pr please.

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2017-07-26_09:36:13-rados-wip-kefu-testing2-distro-basic-smithi/
- http://pulpito.ceph.com/kchai-2017-07-26_14:52:23-rados-wip-kefu-testing2-distro-basic-smithi/

**Comment by tchaikov** [comment]
was #16565

**Pull request merged**